### PR TITLE
Fast abi benchmarks

### PR DESCRIPTION
**Pull request by ujjwalchadha**

### CODE REVIEW

File: BenchmarkComponent/BenchmarkComponent.idl [hunk 1]
```diff
@@ -118,6 +118,35 @@ namespace BenchmarkComponent
...
+    [version(1.0)]
+    unsealed runtimeclass ClassWithFastAbi : NonDefaultFastAbiInterface
+    {
+        ClassWithFastAbi();
+        Int32 DefaultIntProperty { get; };
+    }
+
+    [exclusiveto(ClassWithFastAbi)]
+    [version(2.0)]
+    interface NonDefaultFastAbiInterface
```
Comment (manodasanW) [inline BenchmarkComponent/BenchmarkComponent.idl:132]: Should we also have a non exclusive interface?

Comment (ujjwalchadha) [reply BenchmarkComponent/BenchmarkComponent.idl:132]: Non exclusive interface does not support Fast Abi

Comment (manodasanW) [reply BenchmarkComponent/BenchmarkComponent.idl:132]: Right, but do we want benchmarks for them on a fast abi declared type or do we expect it to be the same as a non fast abi type?

Comment (ujjwalchadha) [reply BenchmarkComponent/BenchmarkComponent.idl:132]: Those would be the same as the non fast abi type

**Review by manodasanW — APPROVED** [review]

**Pull request merged**